stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.12
-version: 1.25.0
+version: 1.26.0
```

Comment:
I just merged another PR with this version. Please bump to `1.27.0`.

Reply:
Fixed.